src/twisted/web/server.py

```diff
@@ -522,6 +522,27 @@ def URLPath(self):
         return urlpath.URLPath.fromRequest(self)
 
 
+    def hyperlinkURL(self):
```

Comment:
As @glyph mentioned, I'd just call this `url`, assuming that's not already taken (I think it isn't?)